File: php/02integer/integer11.php
<?php
$N = mt_rand(100, 999);
$a = intval($N / 100);
$b = intval($N / 10) % 10;
$c = $N % 10;
$sum = $a + $b + $c;
$zarb = $a * $b * $c;
echo "Адади серақама: ".$N;
echo "<br>Қисми садӣ: ".$a;
echo "<br>Қисми даҳӣ: ".$b;
echo "<br>Қисми воҳидӣ: ".$c;
echo "<br>Суммаи рақамҳо: ".$sum;
echo "<br>ҲосилиЗарби рақамҳо: ".$zarb;
?>

File: cpp/02integer/Integer11.cpp
#include <iostream>
using namespace std;

int main()
{
	//Task("Integer11");
	int number;
	cin >> number;
	int a = number / 100;
	int b = number / 10 % 10;
	int c = number % 10;
	int sum = a + b + c;
	int zarb = a * b * c;
	cout << sum << zarb;
	
	return 0;
}
